agent/component-test/softwarecontaineragent_unittest.cpp

```diff
@@ -111,19 +112,15 @@ class SoftwareContainerAgentTest: public ::testing::Test
 };
 
 TEST_F(SoftwareContainerAgentTest, CreatAndCheckContainer) {
-    ContainerID id;
-    bool success = sca->createContainer(valid_config, id);
-    ASSERT_TRUE(success);
-    ASSERT_TRUE(sca->getContainer(id) != nullptr);
+    ContainerID id = sca->createContainer(valid_config);
```

Comment:
Maybe using like `ASSERT_NO_THROW(id = sca->createContainer(valid_config));` would be better